src/lang/std/artifactGraph.ts

```diff
@@ -337,11 +340,13 @@ export function getArtifactsToUpdate({
         })
     }
     return returnArr
-  } else if (cmd.type === 'extrude') {
+  } else if (cmd.type === 'extrude' || cmd.type === 'revolve') {
+    const subType = cmd.type === 'extrude' ? 'extrusion' : cmd.type
```

Comment:
Feel free to ignore this comment since it's very much a bike shed thing. But the name change here seems kind of arbitrary and inconsistent. I get that it makes more sense as a noun, as in "extrusion", but wouldn't it then be "revolution"? If it were me, I would probably just set subType to `cmd.type` and move on.

Reply:
I agree with this. It is a one liner to take the `cmd.type` and match the `subType` since I wanted to keep the same constants for extrude when I generalized the variable names to sweep.